clients/fides-js/src/lib/tcf.ts

```diff
@@ -39,17 +39,22 @@ export const generateTcString = async ({
     tcModel.cmpVersion = CMP_VERSION;
     tcModel.consentScreen = 1; // todo- On which 'screen' consent was captured; this is a CMP proprietary number encoded into the TC string
 
+    // Narrow the GVL to say we've only showed these vendors provided by our experience
+    const vendorIds = [
+      ...(experience.tcf_vendor_consents?.map((v) => +v.id) || []),
+      ...(experience.tcf_vendor_legitimate_interests?.map((v) => +v.id) || []),
```

Comment:
ah, Dawn's commit here https://github.com/ethyca/fides/pull/4256/commits/7b86c153cafb9701aef72642d0589badfa802aa1 reminded me that in addition we should filter to just the IDs that are GVL, now that we are supporting AC. I will do that shortly